### DISCUSSION TITLE
Title

### DISCUSSION THREAD

**Discussion by wulfland** [wulfland/AccelerateDevOps#55] [Q&A] [answered]
# Heading 1
## Heading 2

**bold** _lalaal_

```js
var x = y;
```

:+1: :shipit:

**Comment by rajbos** [comment, marked as answer]
Video example here:
![weekend](https://user-images.githubusercontent.com/6085745/138743251-dce74d63-042a-4569-9b28-1bc24998c9e2.gif)

**Comment by wulfland** [comment]
answer

**Reply by dronkoff** [reply]
![HelloThereBabyYodaGIF](https://user-images.githubusercontent.com/24692707/138743639-bf79831e-0864-47bc-bae4-bf65db3dd311.gif)
